lib/baseFactory.js

```diff
@@ -113,6 +112,31 @@ class BaseFactory {
     }
 
     /**
+     * Remove a Model
+     * @method remove
+     * @param  {Mixed}    config               Config object
+     * @return {Promise}
+     */
+    remove(config) {
```

Comment:
This just removes a row by id? Does it not do a clean-up of everything related to it?